scalardl-web-client-sdk.js

```diff
@@ -89,8 +91,8 @@ class ClientService {
    * @return {Promise<ClientServiceResponse>}
    */
   async registerCertificate() {
-    const request = new CertificateRegistrationRequestBuilder()
-        .withCertHolderId(this.certHolderId)
+    const request = new CertificateRegistrationRequestBuilder().withCertHolderId(
```

Comment:
Is this change from lint ?

Reply:
It was but I reverted it as it was better before.